### DISCUSSION TITLE
Improve backend

### DISCUSSION THREAD

**Discussion by SadanandMiskin** [sagnik3788/FlavorFleet#47] [Ideas]
@sagnik3788  I have an opinion regarding the backend code for this project , there are multiple port for each routes , And it would be a hassle for someone to start all ports, instead 

- Create a single port and assign routes
- Redesign the backend code 

Hope If you are ok , I can create an issue and solve , Thanks.

**Comment by sagnik3788** [comment]
@SadanandMiskin yes you can create a pr for this merge `user-server.js` and  `dish-server.js ` in one
